src/containers/Layout/Header.tsx

```diff
@@ -81,10 +81,10 @@ const Header: React.FC<HeaderProps> = ({ onNotificationBadgeClick }) => {
     className="header"
     aria-label="Page Header"
     headerTools={pageToolbar}
-    logo={<React.Fragment>
+    logo={<>
       <img onClick={() => history.push("/")} src={logo} className="logo" alt="DarwinAI Logo" />
-      <span className="logo-text">COVID-Net</span>
-    </React.Fragment>}
+      <button onClick={() => history.push("/")} className="title"><span className="logo-text">COVID-Net</span></button>
```

Comment:
Could we alternatively move the `onClick` into the span and remove the button? It would accomplish the same thing without the need for an extra element to wrap it.

`<span onClick={() => history.push("/")} className="logo-text">COVID-Net</span>`

The `title` class may not be needed if we choose to do this.

Reply:
@colinywu @boris-nguyen what are your opinions on this? Is it more ideal to move the onClick into the span, or is it better practice to only use onClicks on buttons?

Reply:
My opinion is that this is not a proper use of a button element, which should be primarily used as a clickable element for submitting form, initiating certain actions, etc, but not for navigation within or outside of the app.

This title is used for navigation within the app so `<Link>` element should be used instead.
```suggestion
      <Link to="/" className=".........">COVID-Net</Link>
```
Make sure to change the CSS so it works with `<Link>`
https://reactrouter.com/web/api/Link